packages/cssnano/src/__tests__/postcss-merge-longhand.js

```diff
@@ -154,3 +154,38 @@ test(
     'h1{margin:10px 20px 30px 40px}',
     'h1{margin:10px 20px 30px 40px}',
 );
+
+test(   
+    'should merge custom props and dont remove fallbacks',
+    processCss,
+    'h1{padding-top:10px;padding-right:15px;padding-bottom:20px;padding-left:25px;padding-top:var(--variable);padding-right:var(--variable);padding-bottom:var(--variable);padding-left:var(--variable)}',
+    'h1{padding:var(--variable);padding:10px 15px 20px 25px}'
+);
+
+test(
+    'should merge rules with custom props',
+    processCss,
+    'h1{padding-top:var(--variable);padding-right:var(--variable);padding-bottom:var(--variable);padding-left:var(--variable)}',
+    'h1{padding:var(--variable)}'
+);
+
+test(
+    'should not break unmergeable fallbacks with custom props (2)',
+    processCss,
+    'h1{padding:10em;padding:var(--variable)}',
+    'h1{padding:var(--variable);padding:10em}'
```

Comment:
This looks wrong. Surely the output should be the same as the input..?

Reply:
As described in issue https://github.com/ben-eb/cssnano/issues/448
cssnano shouldn't remove fallbacks for custom CSS properties.

As far i understand that exactly what fix should do.

Reply:
Yes, but the correct order should be: 

```
h1{padding:10em;padding:var(--variable)}
```

`var(--variable)` wouldn't be understood by browsers not supporting it, thus falling back on `10em`. In your case, you overwrite `var(--variable) ` with `10em` in all cases.

Reply:
@karapetyan yep, we should have `h1{padding:10em;padding:var(--variable)}` here

Reply:
@andyjansson @evilebottnawi understood, thanks for review :) I’ll fix it soon.

Reply:
@evilebottnawi friendly ping